cpu/sensors.py

```diff
@@ -98,7 +98,10 @@ def test(self):
             except process.CmdError:
                 self.error(
                     'Starting Service Failed. Make sure module is loaded')
-        process.run('yes | sudo sensors-detect', shell=True)
+        det_op = process.run('yes | sudo sensors-detect', shell=True).stdout
```

Comment:
just one minor change can you please pass `ignore_status=True` as one parameter in `process.run` ?